packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/resources/TransformResource.java

```diff
@@ -0,0 +1,47 @@
+package software.uncharted.terarium.hmiserver.resources;
+
+import software.uncharted.terarium.hmiserver.models.modelservice.PetriNet;
+import software.uncharted.terarium.hmiserver.proxies.skema.SkemaRustProxy;
+import software.uncharted.terarium.hmiserver.proxies.modelservice.ModelServiceProxy;
+
+import org.eclipse.microprofile.rest.client.inject.RestClient;
+
+import io.quarkus.security.Authenticated;
+import io.quarkus.security.identity.SecurityIdentity;
+import lombok.extern.slf4j.Slf4j;
+
+import javax.inject.Inject;
+import java.util.List;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.core.Response;
+
+
+@Path("/api/transforms")
+@Authenticated
+@Slf4j
+public class TransformResource {
+	@Inject
+	@RestClient
+	SkemaRustProxy skemaProxy;
+
+	@Inject
+	@RestClient
+	ModelServiceProxy modelServiceProxy;
+
+	@POST
+	@Path("mathml-to-acset")
+	public Response mathML2ACSet(List<String> list) {
+		return skemaProxy.convertMathML2ACSet(list);
+	}
+
+
+	@POST
+	@Path("acset-to-latex")
```

Comment:
We currently have a mix of hyphens and underscores on our resource endpoints... the accepted convention is to use hyphens so I've opened an [task](https://github.com/DARPA-ASKEM/Terarium/issues/862) to standardize on hyphens.  @mwdchang not a task you have to worry about... this note is here in case anyone else notices the hyphens used here :D